### ISSUE TITLE
Prebuilt Binary for GCC 7.3?

### ISSUE THREAD

**Issue by youngpm** [halide/Halide#3197]
Thought it might be worth asking if this is worth adding as a build target, it's what ships with Ubuntu 18.04. 

Unless the 5.3 builds will work out?  I'm assuming I'd encounter ABI problems.   Guess I should just try :) Thanks!

**Comment by 9cvele3** [comment]
I had no problems using 5.3 build on Ubuntu with gcc 7.3. 

However, I needed to use Halide on CentOS. CentOS by default ships with gcc 4.8, but there are [devtools](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-7/) that make it possible to use gcc 7 on CentOS (my app needs to use gcc 7). gcc 7 by itself works fine this way on CentOS, but I needed to build Halide from source on this platform. None of the existing build targets were working on CentOS with gcc 7.

**Comment by youngpm** [comment]
@9cvele3 Thanks for the info; I take it that the prebuilt version for GCC 5.3 also didn't work on centos for you?

**Comment by 9cvele3** [comment]
Yes, gcc 5.3 prebuilt version didn't work on CentOS for me. But again, it worked on Ubuntu with gcc 7.3 without any problems.
